src/docs/src/config/intro.rst

```diff
@@ -126,9 +126,10 @@ by at least one space on each side. This might be useful in the ``[jwt_keys]``
 section, where base64 encoded keys may contain some ``=`` characters.
 
 .. note::
-    In case when you'd like to remove some parameter from the `default.ini`
-    without modifying that file, you may override in `local.ini`, but without
-    any value::
+    In case you'd like to remove some parameter from an `.ini` file
+    (e.g. `default.ini`) without modifying that file, you may override
+    in a subsequently loaded `.ini` file (e.g. `local.ini`), but
```

Comment:
Wonder if `may override it in a...` would sound better?

Reply:
Oops, good catch. Will push a fix.